components/cpx-url/test/cpx-url_spec.js

```diff
@@ -0,0 +1,86 @@
+import { expect } from "@esm-bundle/chai";
+import { elementUpdated, fixture, html } from "@open-wc/testing";
+import { CPXUrl } from "../cpx-url";
+
+describe("CPXUrl", () => {
```

Comment:
I'd love to see some common setup and cleanup using `beforeEach(...)` and `afterEach(...)` to consolidate some of the common element setup if possible.  I realize that it may not make sense since fixtures are being used though.